src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/ChatOptions.cs

```diff
@@ -27,6 +27,9 @@ public class ChatOptions
     /// <summary>Gets or sets the presence penalty for generating chat responses.</summary>
     public float? PresencePenalty { get; set; }
 
+    /// <summary>Gets or sets a seed value used by a service to control the reproducability of results.</summary>
+    public long? Seed { get; set; }
```

Comment:
Unrelated to this change per se, but does specifying a seed override other configuration impacting outputs such as `Temperature`

Reply:
Seed typically doesn't override other settings, it just impacts how the prng is seeded when a prng is used.

Temperature typically impacts how probabilities of next tokens are computed. A model will generate a raw score for each possible next token, and then it'll use a function like softmax to convert those scores into probabilities. Temperature impacts the distribution created by softmax, where a lower value makes the distribution sharper, causing more likely words to have a disproprionately higher probability.